controllers/login.js

```diff
@@ -0,0 +1,37 @@
+const User = require("../model/user");
+
+const LoginController = {
+  async Index(req, res) {
+    res.render("login/index", { message: req.flash("message") });
+  },
+  // eslint-disable-next-line consistent-return
+  async Login(req, res) {
+    try {
+      const { username } = req.body;
+      const { password } = req.body;
+      const user = new User();
+      const isUserAuthenticated = await user.authenticate(username, password);
+      const userId = isUserAuthenticated.id;
+      if (isUserAuthenticated === "incorrect username") {
+        req.flash("message", "Incorrect username");
+        return res.redirect("/login");
+      }
+      if (isUserAuthenticated === "incorrect password") {
+        req.flash("message", "Incorrect password");
+        return res.redirect("/login");
+      }
```

Comment:
This error handling is a bit brittle - Is would be better to have the user class throw and error, and then the flash notices can be set in the catch of the `try.. catch`